adr/0004-event-bus-ownership.md

```diff
@@ -0,0 +1,38 @@
+# 4. Own event bus exchanges and queues in services
+
+Date: 2018-05-17
+
+## Status
+
+Proposed
+
+## Context
+
+In a service-based architecture, infrastructure needs to be owned by a service so that it is provisioned and maintained as a first class citizen.
+
+The [event system](0001-events-system.md) based on RabbitMQ requires the setup of `exchanges` and `queues` that `bind` to them.
+
+Practical experience developing eLife lead to no mutual dependencies between services, with e.g. service B strictly being downstream from service A with no events feeding back into service A.
+
+Some services may require a generic binding to many other exchanges in the interest of observability, e.g. a dashboard aggregating events from many different services.
+
+## Decision
+
+Create an `exchange` for each data type, as part of a service publishing events.
+
+Create one `queue` as part of a service consuming events.
+
+Create more than one `queue` in a service if specialized behavior is required, in opposition to a generic type of consumer process.
```

Comment:
Might be out of scope for Libero itself?

Reply:
Just wanted to make clear it's allowed to integrate like that, but can remove.